### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Veteran Facing Forms - 20-10207

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#72865]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** For the living situation question, the conditionally-revealed follow-up question doesn't follow the [design system guidelines for conditionally-revealed fields](https://design.va.gov/patterns/ask-users-for/relationship#conditionally-revealed-fields-can-be-used-if-the-following-conditions-are-met), since there's an additional info component between the "other" checkbox and the new input. 
   
   I think there's a good case to be made that this is okay, though, and the guidelines should be updated. So long as the next tab stop is (1) not another input and (2) provides relevant information for completing the revealed input (additional info or accordion, basically), I think it makes sense. But it's a little less tidy than the rules we have now.
   
   Alternatively, I think the follow-up question could make sense on its own page --- especially if there's a way around the 100 character limit. People might have unusual circumstances that require a lot of explanation, so giving that question its own page so it can stand on its own seems defensible.
   
   I'm happy with whatever route you want to take!

**Comment by jeana-adhoc** [comment]
This has been addressed in the sketch files.  I'm breaking up the questions and putting the conditionally revealed on its own page. A user who selects "A situation not listed here" will conditionally get a text area to fill in the 'other' situation.
